utils/SwervePoseEstimatorBase.java

```diff
@@ -113,10 +113,15 @@ public void update() {
                 VisionMeasurement m = vision.getVisionMeasurement();
                 EstimatedRobotPose estimation = m.estimation();
                 Pose2d estimatedPose = estimation.estimatedPose.toPose2d();
+                
 
-                poseEstimator.addVisionMeasurement(estimatedPose, estimation.timestampSeconds, m.confidence());
+                poseEstimator.addVisionMeasurement(estimatedPose
```

Comment:
I don't understand why we're changing this file. It's just breaking the line in an odd way (you generally leave the comma on the first line) and adding some empty lines. What's the point of this PR?